src/test/java/qna/domain/AnswersTest.java

```diff
@@ -0,0 +1,27 @@
+package qna.domain;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class AnswersTest {
+
+    @Test
+    void add_answer_test() {
+        Answers answers = new Answers(Answer.create(User.create("deokmoon"), Question.create("hello", "world")));
```

Comment:
도메인모델에 테스트용 픽스처 생성 팩토리메소드가 존재하는걸로 보이네요.
도메인모델은 테스트에 대한 관심이 없어야한다고 생각해요. 🤔
```suggestion
        Answers answers = new Answers(Answer.create(User.create("deokmoon", "password", "name", "deokmoon@gmail.com"), Question.create("hello", "world"), "contents"));
```
이렇게 하거나
테스트에 불필요한 파라미터는 임의로 지정하고싶으시다면

> 1. 내부에 픽스처를 리턴하는 메서드를 생성
> 2. 테스트용 픽스처 생성을 도와주는 클래스 생성
> 3. 테스트 내부에 멤버변수로 테스트 픽스처를 선언하고 `@BeforeEach`를 통해 각 테스트 실행시마다 초기화

등 여러 방법으로 테스트 픽스처를 생성할수 있을것 같아요! 😀